src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -252,14 +377,15 @@ export default class FileField extends React.Component {
               <label
                 id={`${idSchema.$id}_add_label`}
                 htmlFor={idSchema.$id}
-                className="usa-button usa-button-secondary"
+                className="vads-u-display--inline-block"
               >
                 <span
                   role="button"
+                  className="usa-button usa-button-secondary vads-u-padding-x--2 vads-u-padding-y--1"
                   onKeyPress={e => {
```

Comment:
These changes make the `span` the same size as the `label` when it has a focus highlight. It also increases the upload button target size to make it more accessible.